chapter_02B_abstractions.asciidoc

```diff
@@ -631,59 +581,53 @@ def ej_synchronise_dirs(reader, filesystem, source_root, dest_root):
 [source,python]
 [role="skip"]
 ----
+class FakeFileSystem(list): #<1>
```

Comment:
The code here and below don't appear to be in the rendered version of the book - are they meant to be?